build.sbt

```diff
@@ -13,11 +13,11 @@ libraryDependencies ++= Seq(
   "com.gu"                   %% "atom-manager-play"           % atomLibVersion,
   "com.gu"                   %% "atom-publisher-lib"          % atomLibVersion,
   "com.gu"                   %% "configuration-magic-core"    % "1.3.0",
+  "com.gu"                   %  "fezziwig"                    % "0.1.1",
```

Comment:
should be  ` "com.gu"                   %%  "fezziwig"                    % "0.2",`